specification/components/schemas/OperationOutcome.yaml

```diff
@@ -1,5 +1,49 @@
 type: object
-description: Outcome of an operation that does not result in a resource or bundle being returned (e.g. error, async/batch submission).
+description: |
+  Outcome of an operation that does not result in a resource or bundle being returned (e.g. error, async/batch submission).
+  There are a number of possible error codes that can be returned along with a more detailed description in the `display` field.
+
+  There are general outcomes:
+  | Code                       | Response Code | Description                                   |
+  | -------------------------- | ------------- | --------------------------------------------- |
+  | ACCESS_DENIED              | 401           | Used when the user does not have permission for a particular request. eg. when their asid does not have the correct interactions attached to it. |
+  | FAILURE_TO_PROCESS_MESSAGE | 500           | A default message when something really bad has happened that the system could not handle.	|
+  | UNABLE_TO_CALL_SERVICE     | 408           | For a synchronous request, the downstream domain processing has not completed within the configured timeout period. |
+  | UNSUPPORTED_SERVICE        | 400           | The service the user requested an endpoint that does not exist - so is unsupported. eg. /Patient/9999999999/Pets |
+
+  Retrieval outcomes:
+  | Code                       | Response Code | Description                                   |
+  | -------------------------- | ------------- | --------------------------------------------- |
+  | RESOURCE_NOT_FOUND         | 404           | The resource was not found. |
+  | INVALID_RESOURCE_ID        | 400           | The resource id was not valid. For example a NHS Number is presented which is not a valid NHS Number. |
+  | INVALIDATED_RESOURCE       | 404           | The resource has been invalidated so could not be returned. |
+
+  Search outcomes:
+  | Code                       | Response Code | Description                                   |
+  | -------------------------- | ------------- | --------------------------------------------- |
+  | INVALID_SEARCH_DATA	       | 400           | The search parameters are invalid. A description of what exactly is at fault will be added to the display. |
+  | TOO_MANY_MATCHES           | 200           | Too many matches were found - user should be told to refine their search parameters.	|
+
+  Update outcomes:
```

Comment:
isn't there also the same retrieval outcomes if attempt to update an NHS Number which has been invalidated?

Reply:
Moved into `general`